lib/cli/src/frameworks/lit/js/Header.stories.js

```diff
@@ -0,0 +1,16 @@
+import './Header';
+import { html } from 'lit';
+
+export default {
+  title: 'Example/Header',
```

Comment:
I've been meaning to add `parameters: { layout: 'fullscreen' } ` to every template for Header and Page. I think it's really interesting to show people this feature as it's not that known and people always see lots of value when I show it to them, as well as it's better to have a fullscreen layout for elements like header/footer/page. Maybe we can start with this template. What do you think? This comment applies to Header and Page stories

Reply:
I'd say make a single PR that does it everywhere @yannbf